index.js

```diff
@@ -114,38 +115,18 @@ var FloatingLabel  = createReactClass({
   },
 
   render() {
-    var props = {
-        autoCapitalize: this.props.autoCapitalize,
-        autoCorrect: this.props.autoCorrect,
-        autoFocus: this.props.autoFocus,
-        bufferDelay: this.props.bufferDelay,
-        clearButtonMode: this.props.clearButtonMode,
-        clearTextOnFocus: this.props.clearTextOnFocus,
-        controlled: this.props.controlled,
-        editable: this.props.editable,
-        enablesReturnKeyAutomatically: this.props.enablesReturnKeyAutomatically,
-        keyboardType: this.props.keyboardType,
-        multiline: this.props.multiline,
-        numberOfLines: this.props.numberOfLines,
-        onBlur: this._onBlur,
-        onChange: this.props.onChange,
-        onChangeText: this.onChangeText,
-        onEndEditing: this.updateText,
-        onFocus: this._onFocus,
-        onSubmitEditing: this.props.onSubmitEditing,
-        password: this.props.secureTextEntry || this.props.password, // Compatibility
-        placeholder: this.props.placeholder,
-        secureTextEntry: this.props.secureTextEntry || this.props.password, // Compatibility
-        returnKeyType: this.props.returnKeyType,
-        selectTextOnFocus: this.props.selectTextOnFocus,
-        selectionState: this.props.selectionState,
-        style: [styles.input],
-        testID: this.props.testID,
-        value: this.state.text,
-        underlineColorAndroid: this.props.underlineColorAndroid, // android TextInput will show the default bottom border
-        onKeyPress: this.props.onKeyPress
-      },
-      elementStyles = [styles.element];
+    var filteredProps = Object.assign({}, this.props, {style : {}, children: null, inputRef : null});
+    var props = Object.assign({}, filteredProps,
+        {
+            onBlur: this._onBlur,
+            onChangeText: this.onChangeText,
+            onEndEditing: this.updateText,
+            onFocus: this._onFocus,
+            style: [styles.input],
+            value: this.state.text,
+            ref: this.props.inputRef || (_ => {})
```

Comment:
Can you please fix this: `(_ => {}` ?

Reply:
Sorry, I don't understand what needs to be fixed.

`(_ => {})` is a valid javascript literal for a no op function.